server/src/main/java/org/elasticsearch/action/delete/DeleteResponse.java

```diff
@@ -40,7 +40,11 @@ public DeleteResponse() {
     }
 
     public DeleteResponse(ShardId shardId, String type, String id, long seqNo, long primaryTerm, long version, boolean found) {
```

Comment:
Could we remove this ctor entirely? Maybe in a follow up?

Reply:
not sure, it comes as a boolean from the engine. This PR's goal was to avoid passing in a `boolean` when we already have a `Result`, aka when parsing.